app/react-native/package.json

```diff
@@ -28,7 +28,7 @@
     "@storybook/addon-links": "^3.1.2",
     "@storybook/addons": "^3.1.2",
     "@storybook/channel-websocket": "^3.1.2",
-    "@storybook/components": "^3.0.0",
+    "@storybook/react": "^3.0.0",
```

Comment:
Does RN use these components? AFAIK not (if so it should be changed in this PR).